src/interface/src/app/shared/form-message-box/form-message-box.component.scss

```diff
@@ -0,0 +1,50 @@
+:host {
+  .message-box {
+    border-radius: 2px;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+    padding: 12px 10px 12px 10px;
+    background-color: rgba(245, 245, 245, 0.4000000059604645);
+    margin-top: 5px;
+    margin-bottom: 1em;
+    font-family: 'Public Sans', 'Roboto', Arial, Helvetica, sans-serif;
```

Comment:
do you mind grouping the font defintiions somehow (mixin, separate class, etc) with the appropiate name? That way we can globally replace it in the future more easily